.github/CODEOWNERS

```diff
@@ -226,6 +226,7 @@
 
 
 # Global components and filters.
+/core/templates/components/angular-html-bind/* @srijanreddy98
```

Comment:
`/core/templates/components/angular-html-bind/
` should be enough to represnt the files in the dir*

Reply:
Done!